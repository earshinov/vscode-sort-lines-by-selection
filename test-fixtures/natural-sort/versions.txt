[BLAH v15;]
[BLAH v1;]
[BLAH v3;]
[BLAH v6;]
[BLAH v13;]
[BLAH v2;]
[BLAH v10;]
