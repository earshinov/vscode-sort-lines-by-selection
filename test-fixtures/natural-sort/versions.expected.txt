[BLAH v1;]
[BLAH v2;]
[BLAH v3;]
[BLAH v6;]
[BLAH v10;]
[BLAH v13;]
[BLAH v15;]
